src/core/ext/filters/client_channel/resolver/google_c2p/google_c2p_resolver.cc

```diff
@@ -308,11 +310,16 @@ void GoogleCloud2ProdResolver::StartXdsResolver() {
         {"TRAFFICDIRECTOR_DIRECTPATH_C2P_IPV6_CAPABLE", true},
     };
   }
+  std::string server_uri = "directpath-trafficdirector.googleapis.com";
```

Comment:
This does a needless allocation.  Suggest just making this a `const char*`.

Reply:
done